src/components/EntryListItem.vue

```diff
@@ -39,15 +49,42 @@
 			'entry.view_is_active': function(val) {
 				if (val)
 					this.$el.scrollIntoView({
-						block: "end", 
-						inline: "nearest", 
+						block: "end",
+						inline: "nearest",
 						behavior: "smooth"});
 			}
 		},
+		data() {
+		  return {
+              // OTP
+              otp: false,
+              otp_loop: undefined,
+              otp_value: "",
+			  keyHandler: e => {
+                  if ((e.ctrlKey || e.metaKey) && e.key == "/" && !e.altKey && !e.shiftKey)  {
+                      this.copyOtp(e);
+                  }
+			  }
+          }
+		},
+        beforeDestroy(){
+            clearInterval(this.otp_loop)
+        },
 		methods: {
 			details(e) {
 				this.$router.route("/entry-details/" + this.entry.id)
 			},
+            setupOTP(url) {
+                let totp = OTP.parseUrl(url)
+				this.otp = typeof totp.key !== "undefined"
+                let do_otp = () => {
+                    totp.next(code => {
+                        this.otp_value = code;
+                    })
+                }
+                this.otp_loop = setInterval(do_otp, 2000)
```

Comment:
Since we're only showing the OTP code on active entries, I think I would rather move this under the watched property for view_is_active.

So when you navigate onto an entry with OTP, it runs this setupOTP function.  When you navigate off, it runs clearInterval.  

In the case where a user has 100s of entries with OTP credentials, computing them every 2 seconds for every entry in the DB will be a pretty big performance hit.